src/libcollections/slice.rs

```diff
@@ -280,34 +280,34 @@ impl<T> [T] {
         core_slice::SliceExt::first_mut(self)
     }
 
-    /// Returns all but the first element of a slice.
-    #[unstable(feature = "slice_extras", reason = "likely to be renamed")]
+    /// Returns the first and all the rest of the elements of a slice.
+    #[stable(feature = "rfc1058", since = "1.3.0")]
```

Comment:
This is a fairly significant change in terms of functionality so I don't think we should insta-stabilize these methods (e.g. leave them as `#[unstable]` for a little longer). Could you also give it a more descriptive feature name than "rfc1058"? Just to help error messages and such later on down the road.

Finally, could you leave the old `tail` methods and such, but `#[deprecate]` them in favor of these new methods?